pandas/io/excel/_openpyxl.py

```diff
@@ -29,12 +29,17 @@ class OpenpyxlWriter(ExcelWriter):
     engine = "openpyxl"
     supported_extensions = (".xlsx", ".xlsm")
 
+    # Because this is used a few times in the class,
+    # it's declared as a variable
+    # Maybe move to within __init__?
+
```

Comment:
Forgot to remove this, I ended up moving the variable into `__init__`. Should it remain there?